hatch_build.py

```diff
@@ -0,0 +1,10 @@
+import typing as t
+
+import django.core.management
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class CustomBuildHook(BuildHookInterface):
+    def initialize(self, version: str, build_data: dict[str, t.Any]) -> None:
+        django.core.management.call_command("compilemessages")
```

Comment:
To be clear, I am uncomfortable with the dependency introduced by this instruction. The only reason to do this was validating the workflow. But if community is ok with removing *.mo files and compiling them at wheel build time and replacing poetry with hatch, I'll invest some effort to rewrite this with babel or gettext.